bci_tester/data.py

```diff
@@ -350,7 +350,11 @@ def create_BCI(
     ):
         marks.append(pytest.mark.__getattr__(build_tag_base.replace(":", "_")))
 
-    if OS_VERSION == "tumbleweed":
+    if TARGET == "manual":
+        baseurl = os.getenv("CONTAINER_URL")
+        if not baseurl:
+            raise ValueError("Missing CONTAINER_URL for TARGET manual")
```

Comment:
this needs to be allowlisted in `tox.ini` . 

how is this different to TARGET=custom with BASEURL=xxx ?

Reply:
With `TARGET=custom` and `BASEURL=foobar` BCI-Test will still apply suffixes on the BASEURL.